File: A2-q2.c
//Program to reate two vectors of type A=(a1,a2,a3) and B=(b1,b2,b3) with numbers of your choice
//(but not random numbers) in the code itself and find A + B and A.B(dot product).

#include <stdio.h>
void main()
{
	int vectA[3] = { 3,5,8 };
	int vectB[3] = { 2,9,6 };
	int sum[3], dotp=0;

	for (int i = 0; i < 3; i++)
	{
		dotp += (vectA[i] * vectB[i]);
		sum[i] = vectA[i] + vectB[i];
	}

	printf("The two vectors include :\nVector A : [");
	for (int i = 0; i < 3; i++)
		printf("  %d  ", vectA[i]);
	printf("]\nVector B : [");
	for (int i = 0; i < 3; i++)
		printf("  %d  ", vectB[i]);
	printf("]\nThe sum of the vectors is : [");
	for (int i = 0; i < 3; i++)
		printf("  %d  ", sum[i]);
	printf("]\nThe dot product is : %d\n", dotp);
}

File: A2_q2.py
#Program to create two vectors of type A=(a1,a2,a3) and B=(b1,b2,b3) with numbers of your choice
#(but not random numbers) in the code itself and to find A+B and A.B (dot product).

vectA = [3, 5, 8] 
vectB = [7, -4, 9] 
sum = []
product=[]
dotp=0
for num1,num2 in zip(vectA,vectB):
    sum.append(num1+num2)
    product.append(num1*num2)
    
for num in range(3): 
    dotp = dotp + product[num]

print("The vectors include:\nVector A :")
print("[",*vectA,"]")
print("\nVector B :")
print("[",*vectB,"]")
print("\nThe sum of the two vectors is :")
print("[",*sum,"]")
print("\nThe dot product is :", dotp,"\n")




########OUTPUT#######
#The vectors include:
#Vector A :
#[ 3 5 8 ]
#
#Vector B :
#[ 7 -4 9 ]
#
#The sum of the two vectors is :
#[ 10 1 17 ]
#
#The dot product is : 73
#
